backend/pkg/api/connect/integration/api_suite_test.go

```diff
@@ -122,3 +178,64 @@ func (s *APISuite) httpAddress() string {
 	}
 	return "http://" + s.api.Cfg.REST.HTTPListenAddress + ":" + strconv.FormatInt(int64(s.api.Cfg.REST.HTTPListenPort), 10)
 }
+
+func setupNetwork(name string) (*TestNetwork, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	network, err := testcontainers.GenericNetwork(ctx, testcontainers.GenericNetworkRequest{
+		NetworkRequest: testcontainers.NetworkRequest{
+			Name: name,
+		},
+	})
+	if err != nil {
+		return nil, err
+	}
+	return &TestNetwork{
+		Network: network,
+		Name:    name,
+	}, nil
+}
```

Comment:
Maybe something for the testutil pkg? Would change the func so that it accepts a ctx parameter